onboarding/mentors.md

```diff
@@ -3,4 +3,53 @@ title: Onboarding Responsibilities for Mentors of New Hires
 description: How to be an effective mentor of a new hire
 ---
 
-TBD
+# Mentorship at Artsy
+
+### Historical Context
+- TODO: talk to Orta/Sarah/Joey about how we used to do it.
+- [Alan's post](http://artsy.github.io/blog/2016/09/22/professional-development-at-artsy-engineering/)
+
+## Why
+One of Artsy's core values is that [People are Paramount](#). Artsy's engineering team works to reflect this in practice...
+
+## What makes a good mentor (Keep?)
+- A first stop for stupid questions
+
+## Mentorship from day 0
+- Keep tabs on your mentee's schedule. Their first day will be full of meetings with People Ops but some good tasks for a mentor[:
+  - Introduce yourself
+  - Set a time for an [introductory coffee or tea](#your-first-meeting)
+  - Get in touch with the mentee's manager
+  - (Re-)familiarize yourself with people resources across the company that will be important to your mentee:
```

Comment:
Might be  nice to explicitly link to any relevant resources (like atlas, the onboarding guide, etc.)